mail/models.py

```diff
@@ -0,0 +1,40 @@
+"""
+Models for the mail app
+"""
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from klasses.models import Klass
+
+
+class AutomaticReminderEmail(models.Model):
+    """
+    Stores information for an automatically sent email
+    """
+    PAYMENT_APPROACHING = 'payment_approaching'
+    REMINDER_TYPES = [PAYMENT_APPROACHING]
+    reminder_type = models.CharField(
+        choices=[(reminder_type, reminder_type) for reminder_type in REMINDER_TYPES],
+        default=PAYMENT_APPROACHING,
+        max_length=30,
+        db_index=True,
+    )
+    days_before = models.SmallIntegerField(null=False, blank=False)
+    email_subject = models.TextField(null=False, blank=True)
+    email_body = models.TextField(null=False, blank=True)
+    sender_name = models.TextField(null=False, blank=True)
+
+    def __str__(self):
+        """String representation of AutomaticEmail"""
+        return "AutomaticEmail sender={}, subject={}".format(self.sender_name, self.email_subject)
+
+
+class SentAutomaticEmails(models.Model):
```

Comment:
Can you add a `__str__` method for this class, since it's editable via Django admin?